assets/xml/objects/object_ik.xml

```diff
@@ -1,106 +1,105 @@
-        <Limb Name="object_ik_Standardlimb_013614" Type="Standard" EnumName="OBJECT_IK_LIMB_1C" Offset="0x13614" />
-        <Limb Name="object_ik_Standardlimb_013620" Type="Standard" EnumName="OBJECT_IK_LIMB_1D" Offset="0x13620" />
-        <Skeleton Name="object_ik_Skel_0136A0" Type="Flex" LimbType="Standard" LimbNone="OBJECT_IK_LIMB_NONE" LimbMax="OBJECT_IK_LIMB_MAX" EnumName="object_ik_Limbs" Offset="0x136A0" />
+        <Animation Name="gIronKnuckleVerticalAttackAnim" Offset="0xCE8" />
+        <Animation Name="gIronKnuckleRecoverVerticalAttackAnim" Offset="0x15F8" />
+        <Animation Name="gIronKnuckleEndVerticalAttackAnim" Offset="0x1ABC" />
+        <Animation Name="gIronKnuckleHorizontalAttackAnim" Offset="0x2484" />
+        <Animation Name="gIronKnuckleEndHorizontalAttackAnim" Offset="0x2E7C" />
+        <Animation Name="gIronKnuckleBackHitAnim" Offset="0x367C" />
+        <Animation Name="gIronKnuckleBlockAnim" Offset="0x391C" />
+        <Animation Name="gIronKnuckleDeathAnim" Offset="0x4A04" />
+        <Animation Name="gIronKnuckleFrontHitAnim" Offset="0x5254" />
+        <Animation Name="gIronKnuckleRunAnim" Offset="0x57F4" />
+        <Animation Name="gIronKnuckleWalkAnim" Offset="0x6294" />
+        <Array Name="gObjectIkVtx_009740" Count="566" Offset="0x6730">
+            <Vtx/>
+        </Array>
+        <DList Name="gIronKnuckleArmorFrontDL" Offset="0x9A40" />
+        <DList Name="gIronKnuckleArmorBackDL" Offset="0x9BC8" />
+        <DList Name="gIronKnuckleAxeDL" Offset="0x9D20" />
+        <DList Name="gIronKnuckleRightHandDL" Offset="0x9FD8" />
+        <DList Name="gIronKnuckleLowerRightPauldronDL" Offset="0xA130" />
+        <DList Name="gIronKnuckleLeftHandDL" Offset="0xA1F0" />
+        <DList Name="gIronKnuckleLowerLeftPauldronDL" Offset="0xA348" />
+        <DList Name="gIronKnuckleUpperRightPauldronDL" Offset="0xA408" />
+        <DList Name="gIronKnuckleUpperLeftPauldronDL" Offset="0xA4F0" />
+        <DList Name="gIronKnuckleFrontTorsoArmorMarkingDL" Offset="0xA5D8" />
+        <DList Name="gIronKnuckleRearTorsoArmorMarkingDL" Offset="0xA6D0" />
+        <DList Name="gIronKnuckleRivetsMarking1DL" Offset="0xA780" />
+        <DList Name="gIronKnuckleRivetsMarking2DL" Offset="0xA820" />
+        <DList Name="gIronKnuckleBasMarkingDL" Offset="0xBF80" />
+        <DList Name="gIronKnuckleGerudoHeadDL" Offset="0xC200" />
+        <DList Name="gIronKnuckleHelmetMarkingDL" Offset="0xCF08" />
+        <DList Name="gIronKnuckleTanArmorDL" Offset="0xD038" />
+        <DList Name="gIronKnuckleBronzeArmorDL" Offset="0xD050" />
+        <DList Name="gIronKnuckleSilverArmorDL" Offset="0xD068" />
+        <DList Name="gIronKnuckleBlackArmorDL" Offset="0xD080" />
+        <DList Name="gIronKnuckleBrownArmorDL" Offset="0xD098" />
+        <DList Name="gIronKnuckleWhiteArmorDL" Offset="0xD0B0" />
+        <DList Name="gIronKnuckleGoldArmorDL" Offset="0xD0C8" />
```

Comment:
Are these material DLs? My understanding is that they don't render any geometry; they just color the geometry from other DLs. Is that correct? If so, I think something like `TanArmorMaterialDL` would signal that.